client/state/current-user/gravatar-status/test/selectors.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import set from 'lodash/set';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getCurrentUserTempImage,
+	getCurrentUserTempImageExpiration,
+	isCurrentUserUploadingGravatar,
+} from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#isCurrentUserUploadingGravatar', () => {
+		it( 'returns false when state is undefined', () => {
+			expect( isCurrentUserUploadingGravatar( undefined ) ).to.equal( false );
+		} );
+
+		it( 'returns state when defined', () => {
+			const state = {};
```

Comment:
Let's avoid mutating test data. We can be a bit more redundant in tests, since this makes it easier to read.